### DISCUSSION TITLE
HTML_ROOT

### DISCUSSION THREAD

**Discussion by trobnc** [Daveiano/weewx-wdc#160] [Q&A] [answered]
Right now this skin is set to push files to http://your.ip.address/weewx however, I run multiple skins and those skins are set to go to different HTML_ROOT locations (i.e. weewx/skin1 and weewx/skin2, etc). I see this option is not set up within wdc skin. In the weewx.conf file, [StdReport] >> [[WdcReport]] I've added the HTML_ROOT = /var/www/html/weewx/wdc as the location but it appears some of the other included files are going to the /var/www/html/weewx location. Any way to fix this?

**Comment by trobnc** [comment]
Just to update, I see now in the skin.conf that base path can be updated in the [Extras] section. I've updated this and removed the HTML_ROOT from the WdcReport section of weewx.conf. However, some files are still going to a different base path. I set base_path = /weewx/wdc/ in the skin.conf however the following HTML files are going to different place (note, deb install, html files set for /var/www/html/weewx)
/var/www/html/statistics.html
/var/www/html/celestial.html
/var/www/html/about.html

Any ideas?

**Reply by Daveiano** [reply, marked as answer]
Please remove all files from your output directory when changing the `HTML_ROOT` option in weewx. If there are files left over from a previous report run, the files will simply stay in this location. They won't get removed automatically.

`HTML_ROOT` and `basePath` are two different things. 

If you want the files of the skin to be generated to `/var/www/html/weewx/wdc`, just set the `HTML_ROOT` like said: `HTML_ROOT = /var/www/html/weewx/wdc`

Here is an explanation of `basePath`: https://github.com/Daveiano/weewx-wdc/wiki/Configuration#extras
If you are hosting the skin at http://your.ip.address/weewx/wdc, the `basePath` should be `/weewx/wdc`.

**Reply by trobnc** [reply]
Thank you, removing the files from html_root worked. Current configuration:
weewx.conf -> [StdReport] [[WdcReport]] - I added HTML_ROOT = /var/www/html/weewx/wdc

/skins/wdc/skin.conf -> base_path = /weewx/wdc/

Now all files generated for wdc go to /weewx/wdc

**Reply by Daveiano** [reply]
Ok so it sounds like everything is working now? Glad I could help you!
